clevertap-core/src/test/java/com/clevertap/android/sdk/inapp/evaluation/EvaluationManagerTest.kt

```diff
@@ -40,7 +42,9 @@ class EvaluationManagerTest : BaseTestCase() {
         triggersManager = mockk(relaxed = true)
         limitsMatcher = mockk(relaxed = true)
         storeRegistry = mockk(relaxed = true)
-        evaluationManager = spyk(EvaluationManager(triggersMatcher, triggersManager, limitsMatcher, storeRegistry))
+        templatesManager = mockk(relaxed = true)
+        evaluationManager =
+            spyk(EvaluationManager(triggersMatcher, triggersManager, limitsMatcher, storeRegistry, templatesManager))
```

Comment:
can we remove code smells which are focring us to use spyk() function?

Reply:
A good idea, but let's keep this outside of this review and do it separately.